api/data/it/PD.tsx

```diff
@@ -0,0 +1,431 @@
+                                    Per ricevere l'accordo di adesione, l'ente deve accedere all'
+                                    <Link href="https://selfcare.pagopa.it/auth/login?onSuccess=%2Fonboarding%2Fprod-pn"
+                                        target="_blank"
+                                        rel="noopener noreferrer"
+                                        aria-label=""
+                                        sx={{ color: "primary.contrastText", textDecoration: "underline", fontWeight: "bold" }}
+                                    >
+                                        Area Riservata
+                                    </Link>
+                                    {" "}e seguire i passaggi descritti in{" "}
+                                    <Link href="https://docs.pagopa.it/area-riservata-enti-piattaforma-notifiche/"
+                                        target="_blank"
+                                        rel="noopener noreferrer"
+                                        aria-label=""
+                                        sx={{ color: "primary.contrastText", textDecoration: "underline", fontWeight: "bold" }}
+                                    >
+                                        questa guida
+                                    </Link>
+                                    .
+                                </Typography>
+                                <Typography variant="body2" tabIndex={0} aria-label="" sx={{ color: "primary.contrastText" }} pb={2}>
+                                    Una volta sottoscritto l'accordo in digitale, l'ente dovrà caricarlo e inviarlo a PagoPA S.p.A. sempre dall'Area Riservata integrando i seguenti documenti e moduli debitamente compilati ove richiesto.
+                                </Typography>
+                                <Typography variant="body2" tabIndex={0} aria-label="" sx={{ color: "primary.contrastText" }} pb={2}>
+                                    Per ulteriori informazioni e chiarimenti, è possibile{" "}
+                                    <Link href="/faq"
+                                        target="_blank"
+                                        rel="noopener noreferrer"
+                                        aria-label=""
+                                        sx={{ color: "primary.contrastText", textDecoration: "underline", fontWeight: "bold" }}
+                                    >
+                                        consultare le FAQ 
+                                    </Link>
+                                    {" "}dedicate agli enti.
+                                </Typography>
+                                <Button variant="contained" sx={{ width: "max-content", backgroundColor: "#FFFFFF", color: "#0073E6", '&:hover': { backgroundColor: "#FFFFFF", color: "#0073E6"}}} >
+                                    Aderisci a SEND
```

Comment:
questo button deve portare all'Area Riservata, aggiungere il link

Reply:
@salvatorediocaro il link aggiunto è errato (va su cittadini.notifichedigitali.it) deve andare su https://selfcare.pagopa.it/auth/login?onSuccess=%2Fonboarding%2Fprod-pn